C#/cs01/03.cs

```diff
@@ -4,8 +4,23 @@ namespace Brainchild.Curriculum.Solutions
     //implement bubblesort in C# and also create the necessary tests
     public class BubbleSort
     {
-        public int[] Sort(int[] arr){
-            throw new NotImplementedException();
+        public int[] Sort(int[] arr)
+        {
+		    int temp;
+		    for (int i = 0; i < arr.Length; i++)
+		    {
+			   for (int j =0; j < arr.Length-1; j++)
+			    {
+				   if (arr[j+1] < arr[j])
+				    {
+				    	temp = arr[j];
+					    arr[j] = arr[j + 1];
+					    arr[j + 1] = temp;
+				    }
+			    }
+		    }
+            return arr;
         }
+        
     }
-}
\ No newline at end of file
+}
```

Comment:
Add new line at the end

Reply:
- [ ] i already add it

![image](https://user-images.githubusercontent.com/78071547/113830497-4d2a9880-97a4-11eb-9d7b-af616755beb9.png)